packages/cli/src/run.ts

```diff
@@ -139,7 +139,9 @@ export async function run(
         }
 
         // Validate that all dependents of ignored packages are listed in the ignore list
-        const dependentsGraph = getDependentsGraph(packages);
+        const dependentsGraph = getDependentsGraph(packages, {
+          workspaceVersionsOnly: config.workspaceVersionsOnly === true
```

Comment:
any particular reason behind this explicit comparison to `true` here? it hasn't been done the same way at other call sites, for example here: https://github.com/atlassian/changesets/pull/507/files?file-filters%5B%5D=.md&file-filters%5B%5D=.ts#diff-033b40b5770c213b81d0ae2fe63d820c6595831c6b8d8af5adb1754399409c1fR104

```suggestion
          workspaceVersionsOnly: config.workspaceVersionsOnly
```